src/Nox.Types.EntityFramework/Types/PhoneNumber/PhoneNumberDatabaseConfigurator.cs

```diff
@@ -0,0 +1,29 @@
+﻿
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Nox.Solution;
+using Nox.Types.EntityFramework.Abstractions;
+
+namespace Nox.Types.EntityFramework.Types;
+public class PhoneNumberDatabaseConfigurator : INoxTypeDatabaseConfigurator
+{
+    public NoxType ForNoxType => NoxType.PhoneNumber;
+
+    public virtual bool IsDefault => true;
+
+    public void ConfigureEntityProperty(NoxSolutionCodeGeneratorState noxSolutionCodeGeneratorState, EntityTypeBuilder builder, NoxSimpleTypeDefinition property, Entity entity, bool isKey)
+    {
+        builder
+            .Property(property.Name)
+            .IsRequired(property.IsRequired)
+            .HasMaxLength(30)
+            .HasConversion<PhoneNumberConverter>();
+    }
+
+    public string GetKeyPropertyName(NoxSimpleTypeDefinition key) => key.Name;
+
+    public virtual string? GetColumnType()
```

Comment:
We do not need this method

Reply:
@gjorgji-grgovski-authoritypartners I updated the PR, is it ok to merge?